src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -314,6 +319,19 @@ describe('DetailsFieldsComponent', () => {
     expect(component.saveEntry.emit).toHaveBeenCalledWith(data);
   });
 
+  it('onSubmit an entry without change hours, should not modify the start_date and end_date ', () => {
+    component.entryToEdit = {...entryToEdit, description: 'test', };
+    fixture.componentInstance.ngOnChanges();
+
+    const startHourValue = moment().subtract(3, 'hours').format('HH:mm');
+    const endHourValue = moment().format('HH:mm');
```

Comment:
not sure if this should be the best approach for setting the start and end hours values, as I understand, you are setting those values in the `before each` section. Could you read the current entry "hours" and setting the test variables with those values instead?

Reply:
I have also deleted those hours, now is in the test those values.